### PR TITLE
Adds zolex doctrine encryption

### PR DESCRIPTION
**Pull request by aeimer**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by aeimer** [comment]
@zolex Could you review this and change the type of the project from `library` to `symfony-bundle`?

File: zolex/doctrine-encrypt-bundle/4.0/config/packages/zolex_doctrine_encryption.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ambta_doctrine_encrypt:
+    # Secret Key path
+    # secret_directory_path: '%kernel.project_dir%'   # Default value
```
Comment (Nyholm) [inline zolex/doctrine-encrypt-bundle/4.0/config/packages/zolex_doctrine_encryption.yaml]: Please remove "default config" that are in comments. 
You should give a link to the docs instead.

File: zolex/doctrine-encrypt-bundle/4.0/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                                                          </>
+<bg=blue;fg=white> You now need to uncomment the wanted algorithm in the    </>
```
Comment (Nyholm) [inline zolex/doctrine-encrypt-bundle/4.0/post-install.txt]: This is not true, right?

It is already selected on (which is good)

Comment (aeimer) [reply zolex/doctrine-encrypt-bundle/4.0/post-install.txt]: I've adapted this issue

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Im happy to merge this as soon as the review bot is happy. Right now you need to change the package type to "symfony-bundle". 

Btw, this package looks abandoned. Should we close this PR?

**Comment by aeimer** [comment]
@Nyholm as I cant adapt this, let us close this PR

**Comment by zolex** [comment]
sorry for answering so late. please refer to the repository I forked this one from. I did nothing relevant...

**Pull request closed without merging**